pyomo/contrib/multistart/high_conf_stop.py

```diff
@@ -5,6 +5,7 @@
 range, given some confidence.
 
 """
+
 from __future__ import division
```

Comment:
This is tangential to this PR but I thought we removed all these `from __future__ import division` statements after we dropped Python 2.7 support.

Reply:
Apparently not. They are all over the place in contrib, core, gdp, dae, etc.